zip-0000.rst

```diff
@@ -163,51 +166,91 @@ SHOULD be added to the ZIP metadata indicating the original authorship
 ZIP Editors
 -----------
 
-The current ZIP Editors are Daira Emma Hopwood, representing the Electric Coin
-Company, and Deirdre Connolly, representing the Zcash Foundation. Both
-can be reached at zips@z.cash . The current design of the ZIP Process
-dictates that there are always at least two ZIP Editors: one from the
-Electric Coin Company and one from the Zcash Foundation. Additional Editors may
-be selected by consensus among the current Editors.
+The current ZIP Editors are:
+- Daira Emma Hopwood and Jack Grigg (representing the Electric Coin
+Company)
+- teor and Conrado Gouvea (representing the Zcash Foundation).
 
+All can be reached at zips@z.cash. The current design of the ZIP Process
+dictates that there are always at least two ZIP Editors: at least one from the
+Electric Coin Company and at least one from the Zcash Foundation.
 
-ZIP Editor Responsibilities & Workflow
---------------------------------------
+Additional Editors may be selected by consensus among the current Editors.
```

Comment:
Do we want to make this "consensus minus one"?

Reply:
I think this is fine as-is. It doesn't specify a quorum and that's ok.